doc/rados/operations/add-or-rm-osds.rst

```diff
@@ -174,13 +174,13 @@ your new OSD before it can begin receiving data. You may use
 ``service ceph`` from your admin host or start the OSD from its host
 machine.
 
-For Debian/Ubuntu use Upstart. ::
+For Ubuntu Trusty use Upstart. ::
 
 	sudo start ceph-osd id={osd-num}
 
-For CentOS/RHEL, use sysvinit. ::
```

Comment:
If a user needs to refer back to the Hammer docs, it is still available online: http://docs.ceph.com/docs/hammer/rados/operations/operating/